src/components/ChartDialog.vue

```diff
@@ -63,6 +63,11 @@
                     <v-col v-html="station.properties.status" />
                     <v-divider />
                   </v-row>
+                  <v-row justify="center" align="center">
```

Comment:
This is the duplicated from https://github.com/wmo-im/wis2box-ui/blob/misc-updates-2022-02-20/src/components/ChartDialog.vue#L47-L51. What is the intended order?

Reply:
(L47-51 are the same as L66-70)

Reply:
Good catch!  Fixed.